src/System.Private.CoreLib/shared/System/PinnableBufferCache.cs

```diff
@@ -431,15 +433,18 @@ private bool TrimFreeListIfNeeded()
         /// Did we put some buffers into m_NotGen2 to see if we can trim?
         /// </summary>
         private bool m_trimmingExperimentInProgress;
+#pragma warning disable 0649
         /// <summary>
         /// A forced minimum number of buffers.
         /// </summary>
         private int m_minBufferCount;
+#pragma warning restore 0649
```

Comment:
We have this warning disabled for corert

Reply:
Is there a compile error if you remove this? I do not see why this warning needs to be disabled.

Reply:
No there isn't any compile error if we remove it in either of the repos